src/main/resources/templates/base.html

```diff
@@ -21,7 +21,9 @@
       </ul>
 
       <div class="col-md-3 text-end">
-        <button type="button" class="btn btn-outline-primary me-2">Login</button>
+        <form action="#" th:action="@{/login/signin}"  method="get">
+          <button type="submit" class="btn btn-outline-primary me-2" id="btn-login">Login</button>
+        </form>
         <button type="button" class="btn btn-primary">Sign-up</button>
```

Comment:
It would be good to put all the buttons inside of the form to not break the page